docs/Reference/CLI/CLI-Subcommands.md

```diff
@@ -442,6 +442,8 @@ Possible values are:
 | `minimal` | Consensus layer | Test       | Used for local testing and development networks. |
 | `pyrmont` | Consensus layer | Test       | Multi-client testnet.                            |
 | `prater`  | Consensus layer | Test       | Multi-client testnet.                            |
+| `kiln`    | Consensus layer | Test       | Multi-client testnet.                            |
+| `ropsten` | Consensus layer | Test       | Multi-client testnet.                            |
```

Comment:
We also have gnosis, swift and less-swift available. This should the same list of networks as what Teku has listed.

Reply:
`swift` and `less-swift` are omitted from the Teku docs because they're only useful for internal acceptance tests. I will omit them here as well.